### PR TITLE
CVE-2019-14884 update

### PR DESCRIPTION
**Pull request by darunesh-RH**

### CODE REVIEW

File: 2019/14xxx/CVE-2019-14884.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        ]
+    },
+    "references": {
+        "reference_data": [
+            {
+                "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14884",
+                "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2019-14884",
+                "refsource": "CONFIRM"
+            }
+        ]
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14884.json]: Please add a reference to upstream security advisory:

 https://moodle.org/mod/forum/discuss.php?d=393587#p1586751

File: 2019/14xxx/CVE-2019-14884.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in Moodle 3.7 to 3.7.2, 3.6 to 3.6.6, 3.5 to 3.5.8 and earlier, where a reflected XSS possible from some fatal error messages."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14884.json]: Please change the description to mention fixed version:

 3.7.3, 3.6.7 and 3.5.9

**Comment by cve-team** [comment]
Just waiting on implementation of iamleot's comments Dhananjay and we'll get that merged. Thanks!

**Comment by cristina479** [comment]
Red Hat CNA,

Can you address the comments in this PR in order to publish this CVE ID to the CVE List?

Thank you,
Cristina

File: 2019/14xxx/CVE-2019-14884.json (hunk 1 above)
Comment (darunesh-RH) [reply 2019/14xxx/CVE-2019-14884.json]: Updated flaw with asked reference.

File: 2019/14xxx/CVE-2019-14884.json (hunk 2 above)
Comment (darunesh-RH) [reply 2019/14xxx/CVE-2019-14884.json]: Updated description with the missing information.

File: 2019/14xxx/CVE-2019-14884.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                "name": "https://moodle.org/mod/forum/discuss.php?d=393587#p1586751",
+                "url": "https://moodle.org/mod/forum/discuss.php?d=393587#p1586751"
+            }
+        ]
+    },
+    "description": {
+        "description_data": [
+            {
+                "lang": "eng",
+                "value": "A vulnerability was found in Moodle through 3.7 to 3.7.2 before 3.73, 3.6 to 3.6.6 before 3.6.7 and 3.5 to 3.5.8 before 3.5.9, where a reflected XSS possible from some fatal error messages."
```
Comment (iamleot) [inline 2019/14xxx/CVE-2019-14884.json]: It seems there is a copypasto.
Please change from:

````
[...] through 3.7 to 3.7.2 before 3.73, 3.6 to 3.6.6 before 3.6.7 and 3.5 to 3.5.8 before 3.5.9 [...]
````

to:

````
[...] versions 3.7 before 3.73, 3.6 before 3.6.7 and 3.5 before 3.5.9 [...]
````

**Pull request merged**